### PR TITLE
[dom-mc] Update 7.0.UP00-19.06-mc

### PR DESCRIPTION
**Pull request by twrobinson**

### CODE REVIEW

**Comment by twrobinson** [comment]
Deleted julia 1.0.4 as it was preventing testing on Dom

File: jenkins-builds/7.0.UP00-19.06-mc [hunk 1]
```diff
@@ -27,8 +27,7 @@
  h5py-2.8.0-CrayGNU-19.06-python3-serial.eb
  IDL-8.5.1.eb                                       --set-default-module
  ipykernel-4.8.2-CrayGNU-19.06-python2.eb
- ipyparallel-6.2.3-CrayGNU-19.06-python3.eb
- julia-1.0.4.eb  
+ ipyparallel-6.2.3-CrayGNU-19.06-python3.eb
```
Comment (teojgo) [inline jenkins-builds/7.0.UP00-19.06-mc]: Remove the space at the end of the line

**Comment by lucamar** [comment]
Retest this please

**Comment by lucamar** [comment]
@twrobinson Please let me remind you that removing the item from the list will not remove the module on the system

**Comment by twrobinson** [comment]
@lucamar I was told the module wasn't on the system and that's why the build was failing.

**Pull request merged**